bubblesub/api/undo.py

```diff
@@ -84,7 +84,6 @@ def __eq__(self, other: T.Any) -> T.Any:
         :return: bool or NotImplemented to fall back to default implementation
         """
         if isinstance(other, UndoState):
-            # pylint: disable=protected-access
```

Comment:
Isn't this needed? Is it smart enough these days to figure `other` is of its own type?

Reply:
It seems so :)